### DISCUSSION TITLE
data update

### DISCUSSION THREAD

**Discussion by josefkozisek** [jomjol/AI-on-the-edge-device#3244] [General]
Hi, Can I manually update some wrong raw data somewhere in data files?

**Comment by SybexX** [comment]
https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-datalogactive
https://jomjol.github.io/AI-on-the-edge-device-docs/data-logging/#data-format

**Comment by josefkozisek** [comment]
Thank you, can be data modify directly from this web application?
![image](https://github.com/user-attachments/assets/752c9356-2910-4c30-be61-46f7dfc943fc)

**Reply by SybexX** [reply]
As far as I know, you have to download the file, edit it on the PC, delete the file from the SD and then upload the edited file again.

**Reply by josefkozisek** [reply]
OMG, Great it works. Thank you.

**Reply by SybexX** [reply]
This allows you to edit the files directly:
for v15.7.0: [AI-on-the-edge-device__update_only_html__v15.7.0.zip](https://github.com/user-attachments/files/17089862/AI-on-the-edge-device__update_only_html__v15.7.0.zip)
for rolling version: [AI-on-the-edge-device__update__rolling_only_html_(098b1bd).zip](https://github.com/user-attachments/files/17089864/AI-on-the-edge-device__update__rolling_only_html_.098b1bd.zip)

![Unbenannt](https://github.com/user-attachments/assets/b9f53fc2-6740-4f47-8879-7940795479ef)

**Reply by josefkozisek** [reply]
Perfect
